src/stores/locks.js

```diff
@@ -37,16 +41,22 @@ exports.setUp = (basePath, BlobStore) => {
 
     unlock (callback) {
       series([
-        (cb) => store.remove(lockFile, cb),
-        (cb) => store.exists(lockFile, (err, exists) => {
-          if (err) return cb(err)
-
-          if (exists) {
-            return cb(new Error('failed to remove lock'))
-          }
-
-          cb()
-        })
+        (cb) => {
+          store.remove(lockFile, cb)
+        },
+        (cb) => {
+          store.exists(lockFile, (err, exists) => {
+            if (err) {
+              return cb(err)
+            }
+
+            if (exists) {
+              return cb(new Error('failed to remove lock'))
+            }
+
+            cb()
+          })
+        }
```

Comment:
:-1: this change makes it harder to read, and is a matter of personal preference